zerver/webhooks/zabbix/doc.md

```diff
@@ -109,3 +109,7 @@ Receive Zabbix notifications in Zulip!
 {!congrats.md!}
 
 ![](/static/images/integrations/zabbix/001.png)
+
+!!! warn ""
+    **Note:** This guide provides the instructions for Zabbix 5.2 and above
+    and for other versions the workflow for setting up a webhook may be different.
```

Comment:
Should this go at the top rather than the bottom?  Maybe just in prose as "This guide is for Zabbix 5.2 and above; older Zabbix versions have a different workflow for creating an outgoing webhook."